packages/core/src/route/createRouter.ts

```diff
@@ -91,19 +96,19 @@ export const performAjvValidation = ({
 		properties: { [config.name]: config.schema },
 		required
 	};
-	const serializedSchema = JSON.stringify(schema);
-	let ajvInstance = ajvCache[serializedSchema];
+	const cacheKey = `${JSON.stringify(schema)}:${JSON.stringify(parameter)}`;
```

Comment:
I wonder if stringifying the object  `{ schema, parameter }` would be preferable, but probably it doesn't matter

Reply:
totally agree, better optimize as much as we can